code/modules/clothing/suits/toggles.dm

```diff
@@ -102,6 +106,9 @@
 	user << "Alt-click on [src] to toggle the [togglename]."
 
 //Hardsuit toggle code
+/obj/item/clothing/suit/space/hardsuit
```

Comment:
It's here to define a variable on the hardsuit so I can keep track of cooldown shit.